src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainSpecLoader.cs

```diff
@@ -223,7 +223,7 @@ private static void LoadTransitions(ChainSpecJson chainSpecJson, ChainSpec chain
             chainSpec.GrayGlacierBlockNumber = chainSpec.Ethash?.DifficultyBombDelays.Count > 5 ?
                 chainSpec.Ethash?.DifficultyBombDelays.Keys.ToArray()[5]
                 : null;
-            chainSpec.ShanghaiBlockNumber = chainSpec.Parameters.Eip1153Transition ?? (long.MaxValue - 1);
+            chainSpec.ShanghaiTimestamp = chainSpec.Parameters.Eip1153TransitionTimestamp ?? (long.MaxValue - 1);
```

Comment:
Other shanghai EIPs? we could add them on shandong branch for sure

Reply:
Will be added with their PRs or by merging shandong